feature-libs/asm/components/asm-main-ui/asm-main-ui.component.ts

```diff
@@ -56,6 +56,7 @@ export class AsmMainUiComponent implements OnInit, OnDestroy {
   isCollapsed$: Observable<boolean> | undefined;
   iconTypes = ICON_TYPE;
 
+  showInactiveCartInfoAlert$: Observable<boolean>;
```

Comment:
Can we instantiate this immediately? 
`this.showInactiveCartInfoAlert$: Observable<boolean> =
        this.asmComponentService.showInactiveCartInfoAlert();`

Reply:
that's ok, even if instantiate this, it will not be used before featureflag 6.2